### PR TITLE
[DAWG] Update of the mandate of the Data Analysis Working Group

### PR DESCRIPTION
**Pull request by dpiparo**

### CODE REVIEW

File: _workinggroups/dataanalysis.md [hunk 1]
```diff
@@ -3,22 +3,8 @@ title: "Data Analysis"
...
-distribution policies, are important. Optimising the actual content of
-analysis formats is a boundary condition to discuss with physics groups,
-but can bring major gains.
+The Data Analysis Working Group aims to eliminate time spent by physicists working on monotonous and laborious tasks while performing physics analysis, and make the publication of physics results efficient both in terms of human and computing resources.  To this end, the group will prioritise capturing the requirements of physics analysis across the HEP domain, aiming to gain a detailed understanding of the needs of physicists by direct consultation. Participation of those people actively involved in writing data analysis software, especially those with experience of publishing several analyses, will be critical to the group’s success. As a close second priority, the group will work with pioneers of technology, both inside and outside of the HEP community. The goal is to build bridges between physics analysis experts and technical innovators, fostering collaboration between these two communities to produce optimal solutions to a well-defined set of problems.
```
Comment (graeme-a-stewart) [inline _workinggroups/dataanalysis.md]: "time spent" -> "the time spent"
"and make" -> "and to make"

Comment (graeme-a-stewart) [reply _workinggroups/dataanalysis.md]: Thanks guys! 

I have one additional criticism, which is that reading this it seems like you're starting from zero. I would add some mention of the work done in the CWP and maybe also the Analysis Ecosystem Review. Some phrase like "building on previous work..." would be enough.

Comment (graeme-a-stewart) [reply _workinggroups/dataanalysis.md]: This is also a mega-line, which makes commenting more awkward. Can you split it up? I would suggest the traditional 80 character warp.

**Comment by graeme-a-stewart** [comment]
Final point - I suggest adding a link to the new Indico page for the WG, https://indico.cern.ch/category/10914/.

**Comment by dpiparo** [comment]
Hi @graeme-a-stewart , thanks for the feedback. I integrated your and other refinements, e.g. added the link to the Google Group of the WG.

**Pull request merged**